homeassistant/components/twitter/notify.py

```diff
@@ -80,6 +82,11 @@ def send_message(self, message="", **kwargs):
                 _LOGGER.warning("'%s' is not a whitelisted directory", media)
                 return
 
+        if target:
+            self.user = target
+        else:
+            self.user = self.default_user
```

Comment:
Instead of juggling with the class variable, we should pass down the right user into the `send_message_callback`.